sql/004_search.sql

```diff
@@ -637,33 +637,33 @@ CREATE OR REPLACE FUNCTION search_tohash(jsonb) RETURNS jsonb AS $$
     SELECT $1 - '{token,limit,context,includes,excludes}'::text[];
 $$ LANGUAGE SQL IMMUTABLE PARALLEL SAFE;
 
-CREATE OR REPLACE FUNCTION search_hash(jsonb) RETURNS text AS $$
-    SELECT md5(search_tohash($1)::text);
```

Comment:
The reason we used the search_tohash function on the search json was so that we can remove the fields that aren't used to drive the context so that when doing things like paging or changing the returned fields, we can just use the cache from ie the previous page.

Reply:
I couldn't see where this function was used so I just decided to apply `search_toash` directly in the `search_hash` call https://github.com/stac-utils/pgstac/pull/54/files#diff-3c3e93de12c078e7f46e264188fcc2ba82eefd77be73795f964ad132d89af285R666